src/sign.builds

```diff
@@ -32,12 +32,14 @@
     <ReadSigningRequired MarkerFiles="@(SignMarkerFile)">
       <Output TaskParameter="SigningMetadata" ItemName="FilesToSign" />
     </ReadSigningRequired>
+    <Message Text="Attempting to sign %(FilesToSign.Identity) with authenticode %(FilesToSign.Authenticode) and strongname %(FilesToSign.StrongName)" />
```

Comment:
it did show up, if you download the artifacts you'll see that the msbuild.log has these. Do we really want this to be on the minimal msbuild log?